### PR TITLE
Feat(Query): CS Kubernetes Node Pool Auto Repair Disabled for Alicloud Terraform

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: CS Kubernetes Node Pool Auto Repair Disabled for Alicloud Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "searchLine":common_lib.build_search_line(["resource", "alicloud_cs_kubernetes_node_pool", name, "management"], []),
+	}
+}
+
+is_equal(secure, target)
+{
+    secure == target
+}else {
+    secure[_]==target
+}
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego:57]:
```suggestion
```

File: assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/metadata.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "id": "81ce9394-013d-4731-8fcc-9d229b474073",
+  "queryName": "CS Kubernetes Node Pool Auto Repair Disabled",
+  "severity": "MEDIUM",
+  "category": "Insecure Configurations",
+  "descriptionText": "Verifies if Alicloud Container Service Node Pool Auto Repair is Enabled",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/metadata.json:6]: Please, improve the `descriptionText` in this query and in the following ones:

- `kics\assets\queries\ansible\gcp\google_container_node_pool_auto_repair_disabled\metadata.json`
- `kics\assets\queries\terraform\gcp\google_container_node_pool_auto_repair_disabled\metadata.json`

File: assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+CxPolicy[result] {
+
+	resource := input.document[i].resource.alicloud_cs_kubernetes_node_pool[name]
+    not common_lib.valid_key(resource, "management")
+	
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_cs_kubernetes_node_pool[%s]",[name]),
+		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego:30]:
```suggestion
        "issueType": "MissingAttribute",
```

File: assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+CxPolicy[result] {
+
+	resource := input.document[i].resource.alicloud_cs_kubernetes_node_pool[name]
+    not common_lib.valid_key(resource.management, "auto_repair")
+	
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_cs_kubernetes_node_pool[%s].management",[name]),
+		"issueType": "IncorrectValue",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego:45]:
```suggestion
        "issueType": "MissingAttribute",
```

File: assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego [hunk 5]
```diff
@@ -0,0 +1,57 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+
+	resource := input.document[i].resource.alicloud_cs_kubernetes_node_pool[name]
+	
+	auto_repair := resource.management.auto_repair 
+    not is_equal(auto_repair, true)
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/query.rego:10]:
```suggestion
   auto_repair == false
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by joaorufi — APPROVED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**